service/staging/staging_controller.py

```diff
@@ -75,23 +76,34 @@ def upload_to_staging():
     The upload endpoint is able to handle single and multiple files provided
     under any key.
 
-    Returns HTTP status code 415 if one of the files' extension is not allowed.
-
     Returns HTTP status code 400 if no files were provided.
+    Returns HTTP status code 200 else
 
-    :return: A JSON object indicating success (e.g. "{ 'success': true }")
+    :return: A JSON object with "content": The files and directories in the staging area.
+            and with "warnings": The warnings if errors occured
     """
-
+    warnings = []
+    file_count = 0
     if request.files:
         for key in request.files:
+
             for file in request.files.getlist(key):
+                file_count = file_count + 1
                 if _is_allowed_file(file.filename):
-                    _upload_file(file)
+                    try:
+                        _upload_file(file)
+                    except Exception as e:
+                        warnings.append({"file_name": file.filename, "success": False, "warning": e.strerror,
```

Comment:
Das ist auch schwer zu lesen. Grundsätzlich hab ich kein Problem damit die Einrückung so zu machen, aber in Fall wäre etwas in der Art besser:

>                         warnings.append({"file_name": file.filename,
>                                          "success": False,
 >                                          "warning": e.strerror,
  >                                          "warning_code": type(e).__name__})